pyproject.toml

```diff
@@ -1,3 +1,53 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "aiobotocore"
+version = "2.13.3"
+description = "Async client for aws services using botocore and aiohttp"
+readme = "README.rst"
```

Comment:
I'd still keep the changes appended... It's possible through dynamic attributes.

Reply:
Thanks for pointing me to setuptools specific dynamic attributes. I will use those to get the versions as well, with reverts much of the other changes related to `__version__`.